evap/staff/templates/staff_user_index.html

```diff
@@ -9,37 +9,39 @@
-    <div class="row mb-3 align-items-center">
-        <div class="col btn-area">
-            <a href="{% url 'staff:user_import' %}" class="btn btn-sm btn-light">{% trans 'Import users' %}</a>
-            <a href="{% url 'staff:user_merge_selection' %}" class="btn btn-sm btn-light">{% trans 'Merge users' %}</a>
-            <a href="{% url 'staff:user_bulk_update' %}" class="btn btn-sm btn-light">{% trans 'Bulk update users' %}</a>
-            <a href="{% url 'staff:user_create' %}" class="btn btn-sm btn-dark">{% trans 'Create new user' %}</a>
-        </div>
-        <div class="col-auto">
-            <div class="btn-switch btn-switch-light">
-                <div class="btn-switch-label">{% trans 'Inactive users' %}</div>
-                <div class="btn-switch btn-group">
-                    <a href="{% url 'staff:user_index' %}?filter_users=false" role="button" class="btn btn-sm btn-light{% if not filter_users %} active{% endif %}">
-                        {% trans 'Show' %}
-                    </a>
-                    <a href="{% url 'staff:user_index' %}?filter_users=true" role="button" class="btn btn-sm btn-light{% if filter_users %} active{% endif %}">
-                        {% trans 'Hide' %}
-                    </a>
+
+    <div class="row mb-3">
+        <div class="col">
+            <div class="card h-100">
+                <div class="card-body">
+                    <h4 class="card-title">{% trans 'User actions' %}</h4>
+                    <ul>
+                        <li class=""><a href="{% url 'staff:user_list' %}" class="">{% trans 'User list' %}</a></li>
+                        <li class=""><a href="{% url 'staff:user_import' %}" class="">{% trans 'Import users' %}</a></li>
+                        <li class=""><a href="{% url 'staff:user_merge_selection' %}" class="">{% trans 'Merge users' %}</a></li>
+                        <li class=""><a href="{% url 'staff:user_bulk_update' %}" class="">{% trans 'Bulk update users' %}</a></li>
+                        <li class=""><a href="{% url 'staff:user_create' %}" class="">{% trans 'Create new user' %}</a></li>
+                    </ul>
                 </div>
             </div>
         </div>
-        <div class="col-3">
-            <div class="input-group">
-                <input type="search" name="search" class="form-control" placeholder="{% trans 'Search...' %}" />
-                <button class="btn btn-light text-secondary" type="button" data-reset="search" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Clear search filter' %}">
-                    <span class="fas fa-backspace"></span>
-                </button>
+        <div class="col">
+            <div class="card h-100">
+                <div class="card-body">
+                    <form method="POST" class="form-horizontal">
+                        {% csrf_token %}
+                            <h4 class="card-title text-center">{% trans 'Search user' %}</h4>
```

Comment:
This one has the `text-center` class, the heading on the other card doesn't. It would be consistent with the staff index page to not use `text-center` (@janno42?)